x-pack/solutions/security/plugins/security_solution/public/app/app.tsx

```diff
@@ -59,19 +62,25 @@ const StartAppComponent: FC<StartAppComponent> = ({ children, history, store, th
               <UserPrivilegesProvider kibanaCapabilities={capabilities}>
                 <ManageUserInfo>
                   <NavigationProvider core={services}>
-                    <ReactQueryClientProvider>
+                    <CasesContext owner={[APP_ID]} permissions={userCasesPermissions}>
```

Comment:
Swapping context around may affect other services. If we do this changes we need to make sure all the functionalities that rely on some provider keep working.

Reply:
Yes, I have tested multiple pages related to the providers for which order has changed and it did not find any issue. Depending on tests as well. But this PR or other PR, this needs to be done if we are using React query.